### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by spencer**
Add dough, yeast and mas_dough_theme + some basic styling

### CODE REVIEW

**Comment by benbarnett** [comment]
Nice one :+1:

File: app/views/admin/partners/_form.html.erb [hunk 1]
```diff
@@ -8,36 +8,36 @@
     <ul>
       <% partner.errors.full_messages.each do |msg| %>
         <li><%= msg %></li>
-      <% end %>
+    <% end %>
```
Comment (neoeno) [inline app/views/admin/partners/_form.html.erb]: Indentation is off on line 11 — there are a few indentation changes in this file that seem unintended

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+9 error node v1.6.2
+10 error npm  v2.7.1
+11 error path /Users/spencer/Work/MAS/WebFeed/webfeeds/package.json
+12 error code ENOPACKAGEJSON
+13 error errno -2
+14 error syscall open
+15 error package.json ENOENT: no such file or directory, open '/Users/spencer/Work/MAS/WebFeed/webfeeds/package.json'
+15 error package.json This is most likely not a problem with npm itself.
+15 error package.json npm can't find a package.json file in your current directory.
+16 verbose exit [ -2, true ]
```
Comment (neoeno) [inline npm-debug.log]: Did you intend to check this in?

**Comment by neoeno** [comment]
Just minor stuff. Looks good :+1:

**Pull request merged**